.github/workflows/docker.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2024 Hemi Labs, Inc.
+# Use of this source code is governed by the MIT License,
+# which can be found in the LICENSE file.
+
+# GitHub Actions workflow to publish Docker images.
+name: "Docker"
+on:
+  push:
+
+concurrency:
+  group: "docker-${{ github.workflow }}-${{ github.event.number || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  # Build and publish Docker image
+  docker:
+    name: "Docker"
+    if: ${{ github.repository_owner == 'hemilabs' }}
```

Comment:
I think `${{` and `}}` can be removed here

Reply:
oh I didn't know that...I'll give that a try